tests/schema/tests.py

```diff
@@ -1651,6 +1651,23 @@ def test_order_index(self):
                 index_name = index_name.upper()
             self.assertIndexOrder(Author._meta.db_table, index_name, ['ASC', 'DESC'])
 
+    @skipIfDBFeature('supports_index_on_text_field')
+    def test_mysql_non_supported_db_index_fields(self):
```

Comment:
This is not a MySQL specific test, therefore it will be better to change name and docstring, e.g.
```python
def test_unsupported_db_index_on_textfield(self):
```